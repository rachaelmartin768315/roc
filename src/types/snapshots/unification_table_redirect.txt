New type variable: {FlexVar(null), rank: 1, mark: NONE, copy: null}
New type variable: {FlexVar(null), rank: 1, mark: NONE, copy: null}
==================== Unification Table ====================
Var 0: {FlexVar(null), rank: 1, mark: NONE, copy: null}
Var 1: {FlexVar(null), rank: 1, mark: NONE, copy: null}
===========================================================
INFO: Perform unification, redirect Var(1) to Var(0)
Unifying 0 <- 1
Redirect check for 0: false
Redirect check for 1: true
  Parent is: 0
==================== Unification Table ====================
Var 0: {FlexVar(null), rank: 1, mark: NONE, copy: null}
Var 1: {FlexVar(null), rank: 1, mark: NONE, copy: null} → 0
Class with root 0:
  0
  1 → 0
===========================================================
