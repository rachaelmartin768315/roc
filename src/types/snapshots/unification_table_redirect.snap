New type variable: {FlexVar(null), rank: 1, mark: none, copy: null}
New type variable: {FlexVar(null), rank: 1, mark: none, copy: null}
==================== START Unification Table ====================
Var 0: {FlexVar(null), rank: 1, mark: none, copy: null}
Var 1: {FlexVar(null), rank: 1, mark: none, copy: null}
==================== END Unification Table ====================
INFO: Perform unification, redirect Var(1) to Var(0)
Unifying 0 <- 1
Redirect check for 0: false
Redirect check for 1: true
  Parent is: 0
==================== START Unification Table ====================
Var 0: {FlexVar(null), rank: 1, mark: none, copy: null}
Var 1: {FlexVar(null), rank: 1, mark: none, copy: null} → 0
Class with root 0:
  0
  1 → 0
==================== END Unification Table ====================
